cg/store/models.py

```diff
@@ -648,6 +648,14 @@ def to_dict(self, parents: bool = False, samples: bool = False, family: bool = F
     def __str__(self) -> str:
         return f"{self.case.internal_id} | {self.sample.internal_id}"
 
+    def get_maternal_id(self, mother: str = None) -> str:
+        """Return parental id."""
+        return mother.internal_id if mother else ""
+
+    def get_paternal_id(self, father: str = None) -> str:
+        """Return parental id."""
+        return father.internal_id if father else ""
```

Comment:
```suggestion
    def get_maternal_sample_id(self) -> str | None:
        """Return parental id."""
        return mother.internal_id if self.mother_links else None

    def get_paternal_sample_id(self) -> str:
        """Return parental id."""
        return father.internal_id if self.father_links else None
```

Would this work? Not sure why we would need to pass a str to get this information